### DISCUSSION TITLE
Stop "current state" node with "reset" value

### DISCUSSION THREAD

**Discussion by redphx** [zachowj/node-red-contrib-home-assistant-websocket#620] [Feature Request]
I want to stop a `current state` node that has "For" configuration.  The `wait until` node accepts `reset` value but the `current state` doesn't.  

Also is it possible to add support for "For" configuration to the `wait until` node too?
  
Thank you.

**Comment by zachowj** [comment]
There's nothing to stop on the current state node. When it receives input it checks the condition at that time and doesn't wait for any values to change. The `for` condition is based on the `last_changed` attributes of the entity.
